### PR TITLE
meta-lmp: update compose apps early start

### PR DESCRIPTION
**Pull request by Tim-Anderson**
The assemble system image to add containers to wic image was updated awhile back to include the docker content.

The early start can take advantage of that and just do a docker compose up similar to the logic for compose-apps.

This updates the early start app to detect and activate the app if it is already loaded into docker.

### CODE REVIEW

**Comment by quaresmajose** [comment]
using `base: compose-apps-early-start: update early start` on the commit title are more alligned with what we usually use.

File: meta-lmp-base/recipes-support/compose-apps-early-start/compose-apps-early-start/compose-apps-early-start [hunk 1]
```diff
@@ -41,14 +41,29 @@ start_restorable_apps() {
...
 
-	aklite-apps run --apps "${shortlist}"
+	if [ -z "${shortlist}" ] ; then
+		all=`ls /var/sota/compose-apps 2> /dev/null`
+		if [ -n "${all}" ] ; then
+			for app in ${all} ; do
+				start_compose_app ${app}
+			done
+		else
+			aklite-apps run --apps "${shortlist}"
```
Comment (mike-sul) [inline meta-lmp-base/recipes-support/compose-apps-early-start/compose-apps-early-start/compose-apps-early-start:62]: iirc, `skopeo` (invoked by aklite-apps`) doesn't inject layers that already exists in the docker store. If so, then there  shouldn't be much difference between `aklite-apps run` and `docker compose up` if images/apps are preloaded to the docker store.

**Comment by Tim-Anderson** [comment]
the problem is skopeo is serial and docker compose does the startup in
parallel

**Comment by ricardosalveti** [comment]
> The assemble system image to add containers to wic image was updated awhile back to include the docker content.
> 
> The early start can take advantage of that and just do a docker compose up similar to the logic for compose-apps.
> 
> This updates the early start app to detect and activate the app if it is already loaded into docker.

Please add this description as part of the commit message as well.

**Comment by ricardosalveti** [comment]
> the problem is skopeo is serial and docker compose does the startup in parallel

Won't both calls be serialized the same way?

**Comment by Tim-Anderson** [comment]
docker runs the starting of the containers in parallel. The time difference is significant on a single core 32 bit processor

**Comment by Tim-Anderson** [comment]
Added a reply

**Review by ricardosalveti — APPROVED** [review]
LGTM, just please update the commit message with your PR description before merging it.

**Pull request merged**